test/controllers/battleController.test.js

```diff
@@ -0,0 +1,55 @@
+const {
+  expect, app, faker, chai,
+} = require('test/testHelper');
+
+describe('battleController', async () => {
+  describe('createBattle on success', async () => {
+    let data, result;
+    beforeEach(async () => {
+      data = {
+        cryptoName: `${faker.name.firstName()}`,
+        playerID: `${faker.name.firstName()}${faker.random.number()}`,
+        healthPoints: faker.random.number(),
+      };
+      result = await chai.request(app)
+        .post('/api/create-battle')
+        .send(data);
```

Comment:
find battle in DB and check, that it was cteated, it is can be just 1 test

Reply:
done